app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :posts
```

Comment:
- [ ] Let's always not forget about handling removing records. If a user is deleted the app will give error message because it is associated with posts. A popular mechanism in rails to handle this is to use `dependent: :destroy`